justfile

```diff
@@ -51,7 +51,16 @@ DC_USER := env_var_or_default("DC_USER", "opener")
 node-install:
     pnpm i
     just frontend/run i18n:en
-    just frontend/run i18n:copy-test-locales
+
+
+# Set up locales for the frontend
+locales mode="ci":
```

Comment:
Kind of a nit, but `test` rather than `ci` might be clearer. It took me a moment to understand that `production` was still intended to run in `ci`. Because it's all running in CI, names that contrast the intention more clearly would be helpful. test/production works, but maybe `test` and `build` are the most clear to me? Mostly just `ci` could be clearer.

Reply:
`test` makes sense, and much clearer. I didn't think about the fact that the `build-img` step is also running in the CI :)
`production` seems better than `build` because we build the app several times, sometimes with only the English locale (Storybook for the docs site), sometimes with test locales (for the Playwright tests) and sometimes for deploying in production (`build-img`)